cms/static/cms/js/modules/cms.toolbar.js

```diff
@@ -420,11 +420,6 @@ var CMS = window.CMS || {};
                 });
             }
 
-            // if there is a screenblock, do some resize magic
-            if (this.ui.screenBlock.length) {
-                this._screenBlock();
```

Comment:
if method is no longer used, remove the method as well (and related tests if required)